public/components/custom_panels/helpers/utils.tsx

```diff
@@ -397,14 +397,13 @@ export const renderCatalogVisualization = async ({
   const catalogSourceName = catalogSource.split('.')[0];
   const catalogTableName = catalogSource.split('.')[1];
 
-  const defaultAggregation = 'avg'; // pass in attributes to this function
-  const attributes: string[] = [];
+  const defaultAggregation = 'avg';
```

Comment:
this constant is not required anymore as the default aggregation comes through queryMetaData